config/symbols.us.txt

```diff
@@ -887,6 +887,7 @@ PLAYER_rotPivotY = 0x800733FA;
 PLAYER_zPriority = 0x800733FC;
 PLAYER_entityId = 0x800733FE;
 PLAYER_step = 0x80073404;
+PLAYER_step_s = 0x80073406;
```

Comment:
HD and PSP also got this removed, can you please restore the others too?

eg. hd
`PLAYER_step_s = 0x80073406;`

psp
`PLAYER_step_s = 0x91E16AE;`